### PR TITLE
Spring Boot: support constraints.drl

### PR DESCRIPTION
**Pull request by ge0ffrey**
Missing some tests that are available on the quarkus variant:
https://github.com/quarkusio/quarkus/pull/8667
Due to this problem:
https://stackoverflow.com/questions/61287244/applicationcontextrunner-for-testing-our-spring-boot-starter-is-there-a-way-to

### CODE REVIEW

File: optaplanner-integration/optaplanner-spring-boot-autoconfigure/src/main/java/org/optaplanner/spring/boot/autoconfigure/SolverProperties.java [hunk 1]
```diff
@@ -21,6 +21,8 @@
 
 public class SolverProperties {
 
+    public static final String DEFAULT_SCORE_DRL_URL = "scoreRules.drl";
```
Comment (rsynek) [inline optaplanner-integration/optaplanner-spring-boot-autoconfigure/src/main/java/org/optaplanner/spring/boot/autoconfigure/SolverProperties.java:24]: So are we changing this to `constraints.drl`?

Comment (ge0ffrey) [reply optaplanner-integration/optaplanner-spring-boot-autoconfigure/src/main/java/org/optaplanner/spring/boot/autoconfigure/SolverProperties.java]: Yes, it's has been changed

**Comment by ge0ffrey** [comment]
@rsynek ready to review again (and merge hopefully :)

**Comment by rsynek** [comment]
@yurloc if you want to review this, please let me know. Otherwise, I will merge the PR tomorrow.

**Pull request merged**